aws-samples/amazon-chime-sdk-js-device-demo
# Update react-dev-utils to 12.0.0

**Issue #:**

**Description of changes:**
Upgrade `react-dev-utils` from 11.0.4 to 12.0.0 to address the security alarm.

**Testing**

1. How did you test these changes?
   * Test in the forked repo and confirm the security alarms are addressed.
   * Run the application locally and confirm the App works well.
   * Confirm the serverless demo can be deployed. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
